lib/unit-ruby/account_limits.rb

```diff
@@ -0,0 +1,108 @@
+module Unit
+  class AccountLimits < APIResource
+    path '/accounts'
+
+    attribute :ach_debit_daily_limit, Types::Limit, readonly: true
+    attribute :ach_debit_monthly_limit, Types::Limit, readonly: true
+
+    attribute :ach_credit_daily_limit, Types::Limit, readonly: true
+    attribute :ach_credit_monthly_limit, Types::Limit, readonly: true
+
+    attribute :card_daily_withdrawal_limit, Types::Limit, readonly: true
+    attribute :card_daily_deposit_limit, Types::Limit, readonly: true
+    attribute :card_daily_purchase_limit, Types::Limit, readonly: true
+    attribute :card_daily_card_transaction_limit, Types::Limit, readonly: true
+
+    attribute :check_deposits_daily_limit, Types::Limit, readonly: true
+    attribute :check_deposits_monthly_limit, Types::Limit, readonly: true
+
+    def self.resource_path(id)
+      "#{super(id)}/limits"
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def update_resource_from_json_api(data)
+      extract_base_resource_details(data)
+
+      clear_attributes!
+
+      extract_ach_debit_limits(data[:attributes][:ach])
+      extract_ach_credit_limits(data[:attributes][:ach])
+      extract_card_atm_limits(data[:attributes][:card])
+      extract_card_transaction_limits(data[:attributes][:card])
+      extract_check_deposit_limits(data[:attributes][:check_deposit])
+
+      mark_as_clean!
+    end
+    # rubocop:enable Metrics/AbcSize
+
+    def extract_base_resource_details(data)
```

Comment:
Kinda silly, but Rubocop was complaining about too many things. I couldn't get around AbcSize as it is.